src/pages/folders/[folderId].tsx

```diff
@@ -0,0 +1,51 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { useQuery } from "@tanstack/react-query";
+
+import { getFolderInfo } from "@/api/getFolderCRUDApi";
+import CardList from "@/components/CardList/CardList";
+import FolderMaker from "@/components/folder/FolderMaker/FolderMaker";
+import FolderModifier from "@/components/folder/FolderModifier copy/FolderModifier";
+import FolderTagList from "@/components/folder/FolderTagList/FolderTagList";
+import Layout from "@/components/Layout/Layout";
+import LinkAddBar from "@/components/LinkAddBar/LinkAddBar";
+import SearchBar from "@/components/SearchBar/SearchBar";
+
+import styles from "./FolderPage.module.scss";
+
+export default function CustomFolderPage() {
+  const [keyword, setKeyword] = useState("");
+  const router = useRouter();
+  const { folderId } = router.query;
+
+  const { data } = useQuery({
+    queryKey: ["folder-info", folderId],
+    queryFn: () => getFolderInfo(folderId as string),
+    enabled: !!folderId,
+  });
```

Comment:
여기 folderId 타입이 `string | string[] | undefined`라 아래쪽에서 타입에러가 나네요..!
타입을 좁히는것은 어떨까요?
```suggestion
  const folderId = router.query.folderId;

  const { data } = useQuery({
    queryKey: ["folder-info", folderId],
    queryFn: () => getFolderInfo(folderId as string),
    enabled: typeof folderId === "string",
  });

  if (typeof folderId !== "string") {
    return <div>잘못된 경로입니다.</div>;
  }
```

Reply:
와... 이 부분 에러는 나는데 어떻게 고칠지 몰라서 헤맸었는데 이런 방법이 있었네요 감사합니다!!!! 요렇게 수정해보겠습니당!!!